content/spin/serverless-ai-tutorial.md

```diff
@@ -232,6 +232,14 @@ Now let's use the Spin SDK to access the model from our app:
 
 {{ startTab "Rust"}}
 
+> Note: The following Rust source code, for this sentiment analysis example, uses serde. Please add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application.
```

Comment:
My usual hobby horse about cutting back the "notes" and the "pleases" and such like I'm afraid, just hopefully making it a bit crisper.

```suggestion
> The Rust source code for this sentiment analysis example uses the `serde` crate. Add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application:
```

and omg I got to suggest a colon to Tim